packages/core/core-flows/src/cart/workflows/add-to-cart.ts

```diff
@@ -93,6 +94,24 @@ export const addToCartWorkflow = createWorkflow(
       return (data.input.items ?? []).map((i) => i.variant_id).filter(Boolean)
     })
 
+    const setPricingContext = createHook("setPricingContext", {
+      cart,
+      variantIds,
+      items: input.items,
+      additional_data: input.additional_data,
+    })
+
+    const setPricingContextResult = setPricingContext.getResult() as any
```

Comment:
question: What would it take to get rid of `as any`?

Reply:
This is not super important right now, but I was just curious.

Reply:
We will have to settle on the properties the hook must/can return and then define a zod schema to validate those properties.

Reply:
Ah I see, should we set it to a record then? So we can get rid of the `any`

Reply:
Sure